02_repeatString/solution/repeatString-solution.spec.js

```diff
@@ -0,0 +1,39 @@
+const repeatString = require("./repeatString");
+
+describe("repeatString", () => {
+  test("repeats the string", () => {
+    expect(repeatString("hey", 3)).toEqual("heyheyhey");
+  });
+  test.skip("repeats the string many times", () => {
```

Comment:
As asked at https://github.com/TheOdinProject/javascript-exercises/pull/325#issuecomment-1411208821, wouldn't it be better of the solutions all had `test()`, with no `test.skip()`, so one could easily verify that the solutions actually work?

Reply:
That's a really good point, thank you for the suggestion! 🎉